src/ssl.c

```diff
@@ -49797,6 +50464,50 @@ int wolfSSL_BN_mod_add(WOLFSSL_BIGNUM *r, const WOLFSSL_BIGNUM *a,
 
 #ifdef WOLFSSL_KEY_GEN
 
+int wolfSSL_BN_generate_prime_ex(WOLFSSL_BIGNUM* prime, int bits,
```

Comment:
This function is only used by ntp-keygen, which is a utility that's not built by default. The current NTP use case doesn't require it, but I had already implemented this and other BN_* functions, so I've left them here.